source/core/parameters_lagrangian.cc

```diff
@@ -1107,30 +1108,64 @@ namespace Parameters
+          {
+            if (rolling_resistance_torque == "no_resistance")
+              std::cout
+                << "Warning, the \"no_resistance\" entry to the \"rolling "
+                   "resistance torque method\" parameter will be deprecated. "
+                   "Please use \"none\" instead."
+                << std::endl;
+
+            if (rolling_resistance_torque == "constant_resistance")
+              std::cout
+                << "Warning, the \"constant_resistance\" entry to the \"rolling"
+                   " resistance torque method\" parameter will be deprecated."
+                   " Please use \"constant\" instead."
+                << std::endl;
+
+            if (rolling_resistance_torque == "viscous_resistance")
+              std::cout
+                << "Warning, the \"viscous_resistance\" entry to the \"rolling"
+                   " resistance torque method\" parameter will be deprecated. "
+                   "Please use \"viscous\" instead."
+                << std::endl;
+            if (rolling_resistance_torque == "epsd_resistance")
+              std::cout
+                << "Warning, the \"epsd_resistance\" entry to the \"rolling "
+                   "resistance torque method\" parameter will be deprecated. "
+                   "Please use \"epsd\" instead."
+                << std::endl;
+          }
+        if (rolling_resistance_torque == "no_resistance" ||
+            rolling_resistance_torque == "none")
           {
-            rolling_resistance_method = RollingResistanceMethod::no_resistance;
+            rolling_resistance_method = RollingResistanceMethod::none;
           }
-        else if (rolling_resistance_torque == "constant_resistance")
+        else if (rolling_resistance_torque == "constant_resistance" ||
+                 rolling_resistance_torque == "constant")
           {
-            rolling_resistance_method =
-              RollingResistanceMethod::constant_resistance;
+            rolling_resistance_method = RollingResistanceMethod::constant;
           }
-        else if (rolling_resistance_torque == "viscous_resistance")
+        else if (rolling_resistance_torque == "viscous_resistance" ||
+                 rolling_resistance_torque == "viscous")
           {
-            rolling_resistance_method =
-              RollingResistanceMethod::viscous_resistance;
+            rolling_resistance_method = RollingResistanceMethod::viscous;
           }
-        else if (rolling_resistance_torque == "epsd_resistance")
+        else if (rolling_resistance_torque == "epsd_resistance" ||
+                 rolling_resistance_torque == "epsd")
           {
-            rolling_resistance_method =
-              RollingResistanceMethod::epsd_resistance;
+            rolling_resistance_method = RollingResistanceMethod::epsd;
           }
         else
           {
             throw(
               std::runtime_error("Invalid rolling resistance torque method "));
```

Comment:
Can you AssertThrow instead, I am trying to keep it cleaner with AssertTrhwo instead of regular throws everywhere
like AssertThrow(false, ExcMessage()).

Reply:
Yeah that is true